### DISCUSSION TITLE
Plants toxic to cats

### DISCUSSION THREAD

**Discussion by XelaAl** [NAIST-SE/AttractingContributors#483] [General]
I'm going to adopt a cat and I wonder whether I should worry about plants at my home. Can they be toxic to a cat?

**Comment by McJul** [comment]
Cats can definitely suffer from poisoning due to ingesting toxic plants. There are certain plants you should avoid, like spring bulbs, amaryllis, rhododendrons. Look up the whole list in the Internet to be sure your pets are safe!
